src/Microsoft.DotNet.Build.Tasks/GenerateEncodingTable.cs

```diff
@@ -0,0 +1,395 @@
+﻿using Microsoft.Build.Framework;
```

Comment:
Had to:
Update .proj file (add output location)
Add "invalid" preferred entries back in (unpick)
Add file back to src (unpick) and regenerate (mostly white-space only diffs).

So yeah, pretty much.